src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -857,5 +857,105 @@ public static unsafe Half Int16BitsToHalf(short value)
         {
             return *(Half*)&value;
         }
+
+        /// <summary>
+        /// Converts the specified double-precision floating point number to a 64-bit unsigned integer.
+        /// </summary>
+        /// <param name="value">The number to convert.</param>
+        /// <returns>A 64-bit unsigned integer whose bits are identical to <paramref name="value"/>.</returns>
+        [CLSCompliant(false)]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static unsafe ulong DoubleToUInt64Bits(double value)
+        {
+            // Workaround for https://github.com/dotnet/runtime/issues/11413
+            if (Sse2.X64.IsSupported)
+            {
+                Vector128<ulong> vec = Vector128.CreateScalarUnsafe(value).AsUInt64();
+                return Sse2.X64.ConvertToUInt64(vec);
+            }
+
+            return *((ulong*)&value);
+        }
```

Comment:
Can we just delegate to the existing signed implementation rather than duplicating everything? e.g.
```C#
[CLSCompliant(false)]
[MethodImpl(MethodImplOptions.AggressiveInlining)]
public static unsafe ulong DoubleToUInt64Bits(double value) => (ulong)DoubleToInt64Bits(value);
```
?

Reply:
Codegen seems the same, so I'll change them. https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABABgAJiBGAOgCUBXAOwwEt8YaBJFqVp3KzC4A3AFgAUGUq1GLdpwDCEfAAdWAGxhQAytoBuQmKMnTq9Zmw7de/QcJoANABxJxE0wGZKAJnKLyAG9JSXIw8LCAbQBZGAwACwgAEy41DQAKWITk1NUNAHlVNggBGgBBAHMK2FxBfRgeDX5+CoBKAF1QiPDib2okcmZcbAAzGEGNEoryABEIBmAtABUIAFUeDFQAIVYMXCp0pPnF8f1sDQYYVq7uoJvb8NYR8nSdXBgfJ1RuXB0GVVU0AwMCS1wkD26wXBEIeADUYGAMNAqD5nAAeBiTJgVAB85HqYHIAF5yPDEcjUTRFLBsMCdGBztgoKsBKMYOkzhcruVcOsWKh0q13DCHsQAOzkN4fL4oKklepQDArPmbFAchFC+4QgC+IWhIvF5AAVOl0pipkbWgAyTmXTX67q6jwOiIxOKJFJpTLunJpQrFUqVarGOoNJhNJgtDpanp9KgDIZsiZTWbHZZrDbbXa4HyHNOnc52mNhKEisKGs1YtpzBbpzMoHZ7DmFq7Ch5O4vkN3ZT15b093IFIqsEq4cpVGqhxrNbHRl2xmQJ1ljchV1O1mAreuN3B5jf4ltgiGlstPF5Sz6Ob5cX7/QGKkFHssnsvhMlIqAo9FVvEE4mkhEPy/KkaTpBkNCZFlhjGZsuVaHl60FNtX3LCULxlOUmAVJUIEQgl7VfDt51uQ0TXSKtLRtQ9kMdSRtSAA===